typespec-tests/customization/src/main/java/CustomizationEncodeBytes.java

```diff
@@ -22,6 +22,8 @@ public void customize(LibraryCustomization customization, Logger logger) {
         PackageCustomization packageCustomization = customization.getPackage("com.encode.bytes.models");
         ClassCustomization classCustomization = packageCustomization.getClass("Base64UrlArrayBytesProperty");
 
+        classCustomization.getProperty("value").setModifier(Modifier.PRIVATE);
```

Comment:
Is it affect serialization? Or it is just a test on customization.

Reply:
The customization was wrong for stream-style serialization. Since stream-style has knowledge of the model being deserialized it can better generate models using `final` variable (constructor only, read only, constants) whereas Jackson based deserialization the properties are left non-final to allow Jackson to reflectively set them after creating an instance. So, this customization was adding another constructor to the model which had constructor only properties that were `final` and this is just an update to the customization to also remove the final from the property to make it valid code.